docs/api/sql/Function.md

```diff
@@ -1093,8 +1093,8 @@ Example:
 
 ```SQL
 SELECT ST_AsText(
-    ST_Reverse(ST_GeomFromText('LINESTRING(0 0, 1 2, 2 4, 3 6)'))
-) AS geom
+               ST_Reverse(ST_GeomFromText('LINESTRING(0 0, 1 2, 2 4, 3 6)'))
```

Comment:
You changed all indents in the docs. This is NOT acceptable and will break the documentation website. Please only change the place you are supposed to change.